quipucords/scanner/openshift/entities.py

```diff
@@ -155,29 +158,70 @@ def __raise__(self):
         return (self.message,)
 
 
-# update nested model references - this should always be the last thing to run
-_update_model_refs()
-
-
-@dataclass
-class Cpu:
-    """Class representing CPU's attributes."""
+class NodeResources(OCPBaseEntity):
+    """Class representing node's resources."""
+
+    cpu: float = None
+    memory_in_bytes: int = None
+    pods: int = None
+    _kind = "node-resources"
+
+    # https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu
+    @validator("cpu", pre=True)
+    def _convert_cpu_value(cls, value):
+        millicore_suffix = re.compile("^\d+m$")
+        if isinstance(value, str) and millicore_suffix.match(value):
+            value = float(value.replace("m", "")) / 1000
+        else:
+            raise ValueError(CPU_VALUE_ERROR)
```

Comment:
we don't need this error. consider if cpu is not in microcores (or even is an integer or float).

Reply:
Your jugment is correct in both cases. I will modify.